io.catenax.return_request/2.0.0/ReturnRequest.ttl

```diff
@@ -0,0 +1,245 @@
+#######################################################################
+# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft
+# Copyright (c) 2023 Robert Bosch GmbH
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 LRP Autorecycling Leipzig GmbH
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Circular Economy Solutions GmbH (C-ECO)
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the 
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license, 
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
```

Comment:
Should all the version the eclipse-related prefix be changed to 2.1.0 or just necessary for this meta-model?
As far as I understand, for making this change I would have to also upgrade my Aspect-Model-Editor to a new version, otherwise I cannot open the model after the changeis done. Is it correct?

Reply:
correct, all four have to reference 2.1.0 (which will automatically happen if you open and save it in the new version of the editor)
All updated/new models are 2.1.0